src/_locales/en/messages.json

```diff
@@ -310,7 +310,7 @@
     "message": "Choose subscription"
   },
   "flattr_subscription_message": {
-    "message": "Please choose and activate your subscription to flattr creators."
+    "message": "Keep in mind that without an active subscription, your favorite creators won't receive any money from you. Any flattrs made without an active subscription will not be converted into money."
```

Comment:
nit "Any flattrs made without an active subscription will not be converted into money." seems redundant, but it's not a blocker issue for me, we can always remove it later.